mlflow/pytorch/__init__.py

```diff
@@ -363,6 +368,7 @@ def _load_model(path, **kwargs):
     :param kwargs: Additional kwargs to pass to the PyTorch ``torch.load`` function.
     """
     import torch
+    pytorch_version = torch.__version__
```

Comment:
Do we need to define this variable? Can we directly pass `torch.__version__` to `LooseVersion`

Reply:
Done